libraries/core-react/src/SelectionControls/Radio/Radio.tsx

```diff
@@ -46,18 +45,24 @@ const Input = styled.input.attrs(({ type = 'radio' }) => ({
     display: inline;
   }
 `
+type StyledRadioProps = Pick<Props, 'disabled'>
```

Comment:
What is Pick?

Reply:
Typescript has a lot of nifty [utility types ](https://www.typescriptlang.org/docs/handbook/utility-types.html)👌